setup.py

```diff
@@ -13,7 +13,7 @@
 MINOR = 0
 MICRO = 0
 PRERELEASE = "rc1"
```

Comment:
I wonder if we should bump the prerelease version too because rc1 has already been released but installing from this branch would make it look like rc1 hasn't been released.

```suggestion
PRERELEASE = "rc2"
```

Reply:
Yeah that makes more sense to me 🤔 
I have just done it this way cause it is what I have always done lol (probably copied from somewhere else the first time).  I will commit your suggestion and start doing that going forward